src/identityApiClient.ts

```diff
@@ -140,7 +173,7 @@ export default function IdentityAPIClient(this: IIdentityApiClient, mpInstance)
 
         if (mpInstance._Store.identityCallInFlight) {
             invokeCallback(
-                callback,
+                callback as Callback,
```

Comment:
type here is `Callback` but in the function signature it's `IdentityCallback`...I'm pretty sure it should be the latter